sdk/eventhub/Microsoft.Azure.WebJobs.Extensions.EventHubs/src/Utility.cs

```diff
@@ -42,42 +42,19 @@ private static LogLevel GetLevel(Exception ex)
                 throw new ArgumentNullException(nameof(ex));
             }
```

Comment:
I've simplified this code because some exception patterns from T1 don't happen anymore. We would double check that logging is reasonable as part of https://github.com/Azure/azure-sdk-for-net/issues/16638